lr-cli/cortx_setup/commands/storageset/config/durability.py

```diff
@@ -64,52 +73,71 @@ class DurabilityConfig(Command):
 
     def run(self, **kwargs):
         try:
-            index = 'storage_durability_index'
+            confstore_file_index = 'storage_durability_index'
```

Comment:
Why is `storage_durability_index` mapped to `confstore_file_index`

Reply:
Its not mapped to confstore_file_index its just a way to refer to confstore in this file...
Do you want me to change that variable as it might create confusions..?

Reply:
Yes please change the variable to `confstore_config`